pyanaconda/payload/__init__.py

```diff
@@ -389,6 +389,26 @@ def mirrorEnabled(self):
         """
         return True
 
+    @property
+    def disabledRepos(self):
+        """A list of disabled repos."""
+        disabled = []
+        for repo in self.addOns:
+            if not self.isRepoEnabled(repo.name):
+                disabled.append(repo)
+
+        return disabled
+
+    @property
+    def enabledRepos(self):
+        """A list of enabled repos."""
+        enabled = []
+        for repo in self.addOns:
+            if self.isRepoEnabled(repo.name):
```

Comment:
This line has still the issue.
You can't call `repo.name` from the `self.addOns` output because it returns string not objects.